solutions/observability/get-started/logs-essentials.md

```diff
@@ -0,0 +1,142 @@
+
+:::
+
+:::{tab-item} Cloud
+
+Elastic {{observability}} can collect logs from cloud services through Elastic integrations.
+
+1. Select **Add data** from the main menu and then select **Cloud**.
+2. Select your Cloud provider to view the collection of integrations available for that provider.
+3. Select the integration you want to add.
+4. Select **Add**.
+:::
+
+:::::
+
+::::::
+
+::::::{step} Explore logs in Discover
+
+**Discover** lets you quickly search and filter your log data, get information about the structure of your log fields, and display findings in a visualization. Instead of having to log into different servers, change directories, and view individual files. All your logs are available in a single view.
+
+For more information on exploring your logs in **Discover**, refer to [Explore logs in Discover](/solutions/observability/logs/discover-logs.md)
+::::::
+
+::::::{step} Create your first dashboards
+
+Elastic provides a wide range of prebuilt dashboards for visualizing observability data from a variety of sources. These dashboards are loaded automatically when you install [Elastic integrations](https://docs.elastic.co/integrations). You can also create new dashboards and visualizations based on your data views.
+
+To create a new dashboard, select **Create Dashboard** and begin adding visualizations. You can create charts, graphs, maps, tables, and other types of visualizations from your data, or you can add visualizations from the library. You can also add other types of panels, such as filters and controls.
+
+For more information about creating dashboards, refer to [Create your first dashboard](/explore-analyze/dashboards/create-dashboard-of-panels-with-web-server-data.md).
+
+::::::
+
+::::::{step} Set up alerts
+
+Elastic {{observability}} lets you define rules of different types which detect complex conditions and trigger relevant actions. Elastic {{observability}} can send alerts to email, Slack, and other third-party systems. Refer to [Create and manage rules](/solutions/observability/incident-management/create-manage-rules.md) to get started.
+
+::::::
+
+:::::::
+
+## Related resources
+
+Use these resources to learn more about {{observability}} or get started in a different way.
+
+### Quickstarts
+
+Quickstarts are compact hands-on guides that help you experiment with Elastic {{observability}} features. Each quickstart provides a highly opinionated, fast path to data ingestion, with minimal configuration required.
+
+[Browse the Elastic {{observability}} quickstarts](/solutions/observability/get-started/quickstarts.md) to get started with specific use cases.
+
+### Observability integrations
+
+Many [Elastic {{observability}} integrations](https://www.elastic.co/integrations/data-integrations?solution=observability) are available to collect and process your data. Refer to [Elastic integrations](https://www.elastic.co/docs/reference/integrations) for more information.
```

Comment:
This is confusing to me. One link to the marketing site and one to the docs. Do we need to link to both?